Assets/MixedRealityToolkit/Definitions/MixedRealityToolkitConfigurationProfile.cs

```diff
@@ -64,6 +67,20 @@ public MixedRealityCameraProfile CameraProfile
             internal set { cameraProfile = value; }
         }
 
+        /// <summary>
+        /// Camera System class to instantiate at runtime.
+        /// </summary>
+        public SystemType CameraSystemType
+        {
+            get { return cameraSystemType; }
+            internal set { cameraSystemType = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Camera System Class to instantiate at runtime.")]
+        [Implements(typeof(IMixedRealityCameraSystem), TypeGrouping.ByNamespaceFlat)]
+        private SystemType cameraSystemType;
```

Comment:
>cameraSystemType [](start = 27, length = 16)

Will this not be better placed in the camera profile? The way I see it, the less information about the camera that leaks into the general MRTK profile the better.

Reply:
No, it needs to be here, just like all the rest of the service types.

Reply:
IMO all the service types should be moved into their respective profiles. But this is how it's done right now.